src/ngraph_builder.cc

```diff
@@ -41,6 +41,36 @@ const static std::map<tf::DataType, ngraph::element::Type> TF_NGRAPH_TYPE_MAP =
      {tf::DataType::DT_UINT16, ng::element::u16},
      {tf::DataType::DT_BOOL, ng::element::boolean}};
 
+// Helper for storing ops in ng_op_map
+static void SaveNgOp(Builder::OpMap& map, std::string key, 
+                     shared_ptr<ng::Node> value, 
+                     size_t index = 0) { 
+  if (map[key].size() <= index) {
```

Comment:
Why not just do map[key].push_back(value)?

Reply:
Ok - in that case you can lookup the map first - without the use of the `[` based lookup. Then use the push_back.